server/front/package.json

```diff
@@ -8,13 +8,13 @@
     "build": "heft build",
     "build:watch": "tsc",
     "lint:fix": "eslint --fix src",
-    "bundle": "esbuild src/__start.ts --define:process.env.MODEL_VERSION=$(node ../../models/all/lib/__showversion.js) --bundle --minify --platform=node > bundle.js & rm -rf ./dist && cp -r ../../dev/prod/dist . && cp -r ../../dev/prod/public/* ./dist/ && rm ./dist/config.json",
+    "bundle": "esbuild src/__start.ts --define:process.env.MODEL_VERSION=\\\"0.6.4\\\" --bundle --minify --platform=node > bundle.js & rm -rf ./dist && cp -r ../../dev/prod/dist . && cp -r ../../dev/prod/public/* ./dist/ && rm ./dist/config.json",
```

Comment:
Could you please reverse this change. Thanks. Did $() still not working on windows?

Reply:
Oh, sorry about it. Yes, still not working